crypto/rsa/rsa_lib.c

```diff
@@ -9,11 +9,12 @@
 
 #include <stdio.h>
 #include <openssl/crypto.h>
+#include <openssl/core_names.h>
+#include <openssl/engine.h>
+#include <openssl/evp.h>
 #include "internal/cryptlib.h"
 #include "internal/refcount.h"
 #include "crypto/bn.h"
-#include <openssl/engine.h>
-#include <openssl/evp.h>
```

Comment:
Is the move up of these headers purely cosmetic, or was there a practical reason for it?

Reply:
Cosmetic. I've taken to trying to put includes from most general to most specific, i.e.
System includes
Public OpenSSL includes
OpenSSL "internal" includes
OpenSSL "crypto" includes
Local includes